### DISCUSSION TITLE
Suggestion/FeatureRequest: Clustering / Dividers for better sorting

### DISCUSSION THREAD

**Discussion by SHU-red** [linuxserver/Heimdall#709] [Ideas]
I would like to have the opportunity to cluster applications without needing to klick through tags.
If i use tags, i have to click through several levels to get to the target.

Proposal:
- New option in settings: "Cluster Applications by Tags"
- This groups the applications by their tags and as heading for each group, the Tag name should be written
- This way it is also possible to have application in multiple groups

**Comment by KodeStar** [comment]
Something along these lines was implemented in v2.6.0 https://github.com/linuxserver/Heimdall/discussions/1292
